guides/mbed-org.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: guide
+---
+
+# First-time setup
+
+To start developing for the mbed-enabled NRF51-DK, you first need to register an account on the online compiler at [developer.mbed.org](https://developer.mbed.org) and associate the development board model with your account.
```

Comment:
They've got at least 15 of them for us, hopefully all 30 before the first course. We're picking them up this week